File: src/com/leetcode/Easy/ReverseString.java
package com.leetcode.Easy;

// 	344. Reverse String
public class ReverseString {

    public void reverseString(char[] s) {
        int left, right = s.length - 1;
        for (left = 0; left < right; left++, right--) {
            char temp = s[left];
            s[left] = s[right];
            s[right] = temp;
        }
    }
}

File: JavaScript/Easy/ReverseString.js
/**
 * 344. Reverse String
 * @param {character[]} s
 * @return {void} Do not return anything, modify s in-place instead.
 */
const reverseString = function (s) {
  let low = 0, high = s.length - 1;
  while (low < high) {
    let temp = s[low];
    s[low++] = s[high];
    s[high--] = temp;
  }
};